public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/Configuration.tsx

```diff
@@ -39,11 +37,10 @@ export const Configuration = ({
               {...register('isTimeSelectionEnabled')}
               data-testid={selectors.EnableTimeRangeSwitch}
               onChange={(e) => {
-                trackDashboardSharingActionPerType(
-                  e.currentTarget.checked ? 'enable_time' : 'disable_time',
-                  shareDashboardType.publicDashboard
-                );
                 onChange('isTimeSelectionEnabled', e.currentTarget.checked);
+                reportInteraction('dashboards_sharing_public_time_picker_clicked', {
+                  enabled: e.currentTarget.checked,
+                });
```

Comment:
Same, wouldn't it be more useful to get this information from the DB? And same for the annotations.